src/Override/OverrideBehaviour.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ConfigCat\Override;
+
+/**
+ * Class OverrideBehaviour Describes how the overrides should behave.
+ * @package ConfigCat
+ */
+class OverrideBehaviour
+{
+    /**
+     * With this mode, the SDK won't fetch the flags & settings from the ConfigCat CDN, and it will use only the local
+     * overrides to evaluate values.
+     */
+    const LOCAL_ONLY = 10;
```

Comment:
Én ezt javaslom: When evaluating values, the SDK will not use feature flags & settings from the ConfigCat CDN, but it will use all feature flags & settings that are loaded from local-override sources.